src/sidebar/services/test/annotation-test.js

```diff
@@ -29,7 +29,7 @@ class FakeSearchClient extends EventEmitter {
   }
 }
 
-describe('annotations', () => {
+describe('sidebar/services/annotationService', () => {
```

Comment:
`describe` blocks should either reference the module path (`sidebar/services/annotation`) OR the the name of a function/class etc rather than being a hybrid. In the case where a module has a default export with an unambiguous name its fine to use that name as the top-level `describe` block.